### PR TITLE
add basic FE (next.js + chakra ui) scaffolding

### PR DESCRIPTION
**Pull request by michaelrickert**

### CODE REVIEW

**Comment by michaelrickert** [comment]
ptal @davidbyttow

File: web/pages/_app.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { ChakraProvider } from "@chakra-ui/react";
+import { AppProps } from "next/app";
+
+const MyApp = ({ Component, pageProps }: AppProps) => {
```
Comment (davidbyttow) [inline web/pages/_app.tsx:4]: LearnApp (until better name)

File: web/pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { Button, Text } from "@chakra-ui/react";
+import React from "react";
+
+const HelloWorld = () => {
```
Comment (davidbyttow) [inline web/pages/index.tsx:4]: Hello

File: web/pages/index.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  return (
+    <Button>
+      <Text fontWeight="bold">Hello World</Text>
+    </Button>
+  );
+};
+
+export default HelloWorld;
```
Comment (davidbyttow) [inline web/pages/index.tsx:12]: you should set this up in your system to always delete whitespace and auto add last line if needed (somehow)

**Pull request merged**